starter-code/main.js

```diff
@@ -53,6 +53,187 @@ mongoClient.connect(`mongodb://localhost:27017/crunchbase`, (error, db) => {
             })
             break;
 
+            case "4":
+            // 4.- List by name all companies founded in february of 2004.
+            db.collection('companies').find({$and: [{founded_year:{$eq:2004}},{founded_month:{$eq:2}}]}, { name: 1, _id: 0 }).toArray((error, result) => {
```

Comment:
Esto también funciona y es un poco menos verboso:

`{ $and: [{founded_year: 2004}, {founded_month: 2}] }`